.travis.yml

```diff
@@ -17,6 +14,12 @@ jdk:
 services:
   - docker
 
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
```

Comment:
I have the same feeling, although this is the [official way](https://docs.travis-ci.com/user/docker/#Using-Docker-Compose) of doing it :)